test/functional/legacy/increment_spec.lua

```diff
@@ -720,4 +739,10 @@ describe('Ctrl-A/Ctrl-X on visual selections', function()
       eq({}, nvim.get_vvar('errors'))
     end)
   end
+
+  it('does not drop leading zeroes', function()
+    execute('set nrformats&vi') -- &vi makes Vim compatible
```

Comment:
That line isn't in the original test.  The original test has a SetUp function which performs `set nrformats&vim`.  This is needed because many of the tests modify `'nrformats'`, e.g. `set nrformats+=alpha`, so it needs to be reset to a known state before each test.

Since you're calling `clear()` before each lua test, that will achieve the same effect.